packages/cli/src/platform/accelerate/disable.ts

```diff
@@ -1,12 +1,29 @@
-import { Command } from '@prisma/internals'
+import { arg, Command, isError } from '@prisma/internals'
+
+import { getRequiredParameterOrThrow, platformParameters, platformRequestOrThrow } from '../../helpers'
 
 export class Disable implements Command {
   public static new(): Disable {
     return new Disable()
   }
 
   public async parse(argv: string[]) {
-    await Promise.resolve('todo')
-    return JSON.stringify(argv)
+    const args = arg(argv, {
+      ...platformParameters.project,
+    })
+    if (isError(args)) throw args
+    const token = getRequiredParameterOrThrow(args, ['--token', '-t'], 'PRISMA_TOKEN')
```

Comment:
`token` should be optional parameter right?

How I reason about the authentication check priorities is:
1. If `token` argument is present, read the value and verify the token
2. If `token` is not present, attempt to read credentials from disk
3. If none of the above, throw `Unauthorized`

Reply:
Indeed, if we have `login`, then the state will be persisted somewhere and token is not required

Reply:
Its required for now, when we add login, yep, it should become optional as you described.

Reply:
I'll add TODO.

Reply:
Actually there are four callsites, no central place, we don't need to be reminded to do this anyways.